templates/frontend/components/navigationMenu.tpl

```diff
@@ -0,0 +1,38 @@
+{**
+ * templates/frontend/components/navigationMenu.tpl
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Primary navigation menu list for OJS
+ *}
+
+<nav id="{$navigationMenu->getTitle()}-navMenu" class="{$navClass} navDropdownMenu">
```

Comment:
this needs to be escaped if using the title: `{$navigationMenu->getTitle()|escape}. Same with any HTML attributes used below. I'd also go ahead and make a pass at replacing spaces which might be in the titles as they will cause errors in the IDs.